src/Swarm/App.hs

```diff
@@ -1,12 +1,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 
 -- |
--- Module      :  Swarm.App
--- Copyright   :  Brent Yorgey
--- Maintainer  :  byorgey@gmail.com
---
--- SPDX-License-Identifier: BSD-3-Clause
```

Comment:
We should leave in the `SPDX-License-Identifier`s.  See https://spdx.dev/ids/ .  But they don't necessarily need to go in the Haddock module comment.

We should probably add an appropriate `SPDX-License-Identifier` to the `boolexpr` files as well (depending on whatever the original license is for the `boolexpr` package).

Reply:
I have now added it to all modules.

> they don't necessarily need to go in the Haddock module comment.

Haddock module comment is nice because Haddock extracts that metadata into an info box for display.

Reply:
> Haddock module comment is nice because Haddock extracts that metadata into an info box for display.

Ah, OK, nice, I had forgotten that.